hosting/proxy/nginx.prod.conf

```diff
@@ -107,10 +106,6 @@ http {
       proxy_pass      $apps;
     }
 
-    location = /v1/update {
```

Comment:
There's a potential breaking scenario here I think - if someone upgrades using BB CLI and they have a custom NGINX config it won't boot I believe

Reply:
Discussed on Discord, should be safe, this is just a route update.